src/main/java/uk/gov/ons/ctp/integration/rhsvc/service/impl/CaseServiceImpl.java

```diff
@@ -61,7 +61,7 @@ public CaseDTO getLatestValidNonHICaseByUPRN(final UniquePropertyReferenceNumber
           .debug("non HI latest valid case retrieved for UPRN");
       return mapperFacade.map(caseFound.get(), CaseDTO.class);
     } else {
-      log.debug("No cases returned for uprn: " + uprn);
+      log.with("uprn", uprn).warn("No cases returned for uprn");
```

Comment:
Not sure that these should be warnings. At best info in my opinion.
Just hasn't found any cases - woopey doo - who's going to read that?
That's just my personal opinion